models/article-models.js

```diff
@@ -1,5 +1,28 @@
 const db = require("../db/connection");
 
-exports.selectArticles = (article_id) => {
+exports.selectArticlesByID = (article_id) => {
   return db.query(`SELECT * FROM articles WHERE article_id = $1`, [article_id]);
 };
+
+exports.selectAllArticles = () => {
+  return db.query(`
+   SELECT
+   articles.*,
```

Comment:
We don't want to select every column from articles as the `*` syntax will do, we don't want the `body` column to be included in this query